spring-ai-docs/src/main/antora/modules/ROOT/pages/api/chatclient.adoc

```diff
@@ -425,5 +425,54 @@ public Flux<String> chat(String chatId, String userMessageContent) {
             .stream().content();
     }
 }
+----
+
+=== Logging
+
+The `SimpleLoggerAdvisor` is an advisor that logs the `request` and `response` data of the ChatClient. 
+This can be useful for debugging and monitoring your AI interactions.
+
+To enable logging, add the `SimpleLoggerAdvisor` to the advisor chain when creating your ChatClient. 
+It's recommended to add it toward the end of the chain:
```

Comment:
If we recommend this, why do we not add it at the end of the chain in the code below instead of in the middle?